api/types/resource.go

```diff
@@ -59,7 +59,8 @@ type Resource interface {
 
 // ResourceDetails includes details about the resource
 type ResourceDetails struct {
-	Hostname string
+	Hostname     string
```

Comment:
Kind of feels like we should get rid of hostname and just use the hostname as the friendly name for SSH nodes.

Reply:
The hostname is rendered as a separate column in the UI -- I feel like I'd need more context as to why the hostname is displayed this way before making this decision.

Reply:
Okay, so I'm going to remove this in a separate PR so as not to break `e` while I merge this PR and its companion in.